umbraco/Umbraco.CMS.Backoffice
# Feature: Document Type reference localizations

## Description

Adds support for localization of a Document Type's name, description, tab label, group label, and property label, (property descriptions already support UFM, including localizations, e.g. `{#properties_title-desc}`).

Fixes https://github.com/umbraco/Umbraco-CMS/issues/16813.

## Types of changes


## How to test?

- Add localization keys to an existing Document Type for the name and description
  - See [documentation](https://docs.umbraco.com/umbraco-cms/fundamentals/data/defining-content/document-type-localization) or [original issue](https://github.com/umbraco/Umbraco-CMS/issues/16813) for info on how to do this.
- Check the following places in the CMS to make sure that the Document Type's name is localized:
  - Document Type composition modal
  - Document Create menu
  - Document info workspace (in the General panel)
  - Data Type info workspace (in the References panel)
  - Data Type > Collections > "Columns Displayed" field, add a field (from the Document Types)
  - Data Type > Content Picker > "Allow items of type" field (for Content configuration)
  - Template > "Insert" button > "Value" option > "Choose field" (from the Document Types)
